tests/sharding-e2e-test.sh

```diff
@@ -217,7 +217,20 @@ fi
 
 
 # TODO: Try to get the entry via Entry ID (Tree ID in hex + UUID)
-UUID=$($REKOR_CLI get --log-index 2 --rekor_server http://localhost:3000 --format json | jq -r .UUID)
+echo
+echo "Testing /api/v1/log/entries/retrieve endpoint..."
+
+UUID1=$($REKOR_CLI get --log-index 1 --rekor_server http://localhost:3000 --format json | jq -r .UUID)
+UUID2=$($REKOR_CLI get --log-index 3 --rekor_server http://localhost:3000 --format json | jq -r .UUID)
+
+HEX_INITIAL_TREE_ID=$(printf "%x" $INITIAL_TREE_ID)
+HEX_INITIAL_SHARD_ID=$(printf "%x" $SHARD_TREE_ID)
+
+ENTRY_ID_1=$HEX_INITIAL_TREE_ID$UUID1
+ENTRY_ID_2=$HEX_INITIAL_SHARD_ID$UUID2
+
+# -f makes sure we exit on failure
+curl -f http://localhost:3000/api/v1/log/entries/retrieve -H "Content-Type: application/json" -d "{ \"entryUUIDs\": [\"$ENTRY_ID_1\", \"$ENTRY_ID_2\"]}"
```

Comment:
this will fail if we get a non 200 response; should we actually pipe to jq and test that we get 2 entries in return? If redis were to be empty we'd return a HTTP 200 with an empty array AFAIK.

Reply:
good call, made the change and actually caught a bug! thanks 💯